src/cli.yaml

```diff
@@ -0,0 +1,395 @@
+                index: 1
+                multiple: false
+
+    - get-issue-tree-init-hashes:
+        about: Lists all SHA1 hashes of all issues (introducing commit)
+        version: 0.1.0
+        authors:
+            - Matthias Beyer <mail@beyermatthias.de>
+            - Julian Ganz <neither@nut.email>
+        args:
+
+    - list:
+        about: >
+                 List issues.
+                 Issues are listed in the following form:
+                     <hash> (<date when the issue was added, human readable>) <header line>
+                 More functionality may come and the output format may change.
+
+        version: 0.1.0
+        authors:
+            - Matthias Beyer <mail@beyermatthias.de>
+            - Julian Ganz <neither@nut.email>
+        args:
+            - n:
+                help: List only <n> issues
+                multiple: false
+                takes_value: true
+            - long:
+                short: l
+                long: long
+                help: List long (not one line per issue, executes 'git dit show' on each issue
+                multiple: false
+                takes_value: false
+            - abbrev:
+                short: a
+                long: abbrev
+                help: Abbreviate issue hash
+                multiple: false
+                takes_value: false
+
+    - new:
+        about: Create a new bug report
+        version: 0.1.0
+        authors:
+            - Matthias Beyer <mail@beyermatthias.de>
+            - Julian Ganz <neither@nut.email>
+        args:
+            - message:
+                short: m
+                long: message
+                help: Use this as issue message
+                multiple: false
+                takes_value: true
+            - signoff:
+                short: s
+                long: signoff
+                help: Add a 'Signed-off-by' line, with user and email from gitconfig
+                multiple: false
+                takes_value: false
+            - gpgsign:
+                short: S
+                long: gpg-sign
+                help: Add a GPG signature
+                multiple: false
+                takes_value: false
+            - tempfile:
+                short: t
+                long: tempfile
+                help: Use a temporary file at <path> instead of .git/COMMIT_EDITMSG
+                multiple: false
+                takes_value: true
+                value_names:
+                    - path
+            - metadata:
+                short: M
+                long: metadata
+                help: >
+                        Add metadata. Can be passed multiple times.
+                        Metadata is added and removed via key-value pairs.
+                        Available Keys are:
+                            status=<value>
+                            priority=<value>
+                            severity=<value>
+                            assignee=<value>
```

Comment:
We should definitely have long-options for the "standard"-tags.

We could still have an option for adding general-purpose tags (not only ones starting with `Dit-`). This option should, however, not refer to the keys we actually may use within `git-dit`. This would encourage using this option despite existing sane options. Also, it should not be a short option, imo.

A subcommand is not necessary.